### PR TITLE
Update about.jsp with contributors from V5.109 #7712

### LINKED ISSUE #7712: Update about.jsp with contributors from V5.109

### PR DESCRIPTION
**Pull request by thenaesh**
Fixes #7712 

V5.109 was delayed due to a bug (#7707). In the meantime, there were several new contributors.

### CODE REVIEW

**Comment by whipermr5** [comment]
@malayaleecoder is no longer a one-time contributor :)

File: src/main/webapp/about.jsp [hunk 1]
```diff
@@ -2193,6 +2273,14 @@
             </a>
           </td>
         </tr>
+        <tr>
+          <td>@yliang97</td>
```
Comment (damithc) [inline src/main/webapp/about.jsp]: I thought I saw this person replying with the full name.

Comment (chowyb) [inline src/main/webapp/about.jsp]: Confirmed: found [here](https://github.com/TEAMMATES/teammates/pull/7753#issuecomment-315925420).

Comment (thenaesh) [inline src/main/webapp/about.jsp]: He replied this morning. I'll update accordingly.

**Pull request merged**